packages/components/src/SegmentedControl/src/SegmentedControlItem.module.css

```diff
@@ -0,0 +1,121 @@
+.hop-SegmentedControlItem {
```

Comment:
is it possible to delay and/or add a small transition on the button's color change for the font color? it looks like the text disapear when clicking on it